frontend/views/containers/user-settings/AppLogs.vue

```diff
@@ -0,0 +1,147 @@
+<template lang='pug'>
+  .settings-container
+    section.card
+      h2.is-title-3.c-title Application logs
+
+      .c-header
+        fieldset.c-filters
+          legend.sr-only Visible logs
+          label.checkbox
+            input.input(type='checkbox' name='filter' v-model='filter' value='error')
+            i18n Errors
+          label.checkbox
+            input.input(type='checkbox' name='filter' v-model='filter' value='warn')
+            i18n Warnings
+          label.checkbox
+            input.input(type='checkbox' name='filter' v-model='filter' value='debug')
+            i18n Debug
+
+        button.is-small.c-download(@click='downloadLogs')
+          i.icon-download.is-prefix.c-icon
+          i18n Download
+          //- i18n.hide-touch Download
+          //- i18n.hide-desktop Share
+        a(ref='linkDownload' hidden)
+
+      textarea.textarea.c-logs(ref='textarea' rows='12' readonly)
+        | {{ prettyLog }}
+
+      .buttons
+        i18n.link(tag='a' @click='openTroubleshooting') Troubleshooting
+        button.is-small.is-outlined(@click='clearLogs') Clear Logs
+      br
+      br
+      p.has-text-danger DELETE BEFORE MERGE!
+      .buttons
+        button.is-small.is-outlined(@click='createLog("error")') Create error log
+        button.is-small.is-outlined(@click='createLog("warn")') Create warn log
+        button.is-small.is-outlined(@click='createLog("debug")') Create debug log
+</template>
+
+<script>
+import sbp from '~/shared/sbp.js'
+import { CAPTURED_LOGS } from '@utils/events.js'
+import { downloadLogs } from '@view-utils/captureLogs.js'
+export default {
+  name: 'AppLogs',
+  components: {},
+  data () {
+    return {
+      filter: ['error', 'warn', 'debug'],
+      prettyLog: null
+    }
+  },
+  mounted () {
+    this.updateFormLogs()
+    sbp('okTurtles.events/on', CAPTURED_LOGS, this.updateFormLogs)
+  },
+  beforeDestroy () {
+    sbp('okTurtles.events/off', CAPTURED_LOGS.CON, this.updateFormLogs)
+  },
+  watch: {
+    filter: function () {
+      this.updateFormLogs()
+    }
+  },
+  methods: {
+    createLog (type) {
+      console[type](`This is a new log of type: ${type}.`, 'Heres an array to fill ~3Kb:', Array(200).fill(type))
+    },
+    openTroubleshooting () {
+      alert('TODO link redirect. How to do this...')
+    },
```

Comment:
@pieer do you have any idea how we can change the tab in a _clean way_?

Reply:
What do you mean by clean way?

Reply:
I didn't find out a good way to change the UserSettings modal to this specific tab. How would you do it here?